server/Cargo.toml

```diff
@@ -16,8 +18,8 @@ path = "src/main.rs"
 
 [dependencies]
 ya-relay-proto = "0.4.1"
```

Comment:
Don't you need to bump this to `0.4.2`?

Reply:
No, it turns out that changes that were in `ya-relay-proto` were not used in our yagna builds, and the dependencies should all remain the same as in rc1.

The release was created nevertheless so that we can switch to it on master.

Reply:
Note: this will be further addressed in the `Update previously unupdated dependencies` point in https://github.com/golemfactory/golem-internal/issues/633